src/alcotest.ml

```diff
@@ -163,16 +163,20 @@ let output_file t path =
   Filename.concat (output_dir t) (file_of_path path "output")
 
 let mkdir_p path mode =
+  let sep = Filename.dir_sep in
   let rec mk parent = function
   | [] -> ()
   | name::names ->
-      let path = parent ^ "/" ^ name in
+      let path = parent ^ sep ^ name in
       begin try if not (Sys.is_directory path) then
         Fmt.strf "mkdir: %s: is a file" path |> failwith
       with Sys_error _ -> Unix.mkdir path mode end;
       mk path names in
-  match String.cuts ~empty:true ~sep:"/" path with
-  | ""::xs -> mk "/" xs | xs -> mk "." xs
+  match String.cuts ~empty:true ~sep:sep path with
+  | ""::xs -> mk sep xs
+  (* check for Windows drive letter *)
+  | dl::xs when Str.string_match (Str.regexp "[A-Z]:") dl 0 -> mk dl xs
```

Comment:
would it be possible to not depend on `Str` for this? It adds a dependency to a C binding (`-lcamlstr`) and this is not thread-safe. Maybe https://github.com/dbuenzli/fpath/blob/master/src/fpath.ml#L41 is enough here.

Reply:
Sorry I wasn't very clear: you don't need to bring the full dependency to fpath here; just copy the same logic here should be sufficient and would avoid growing the dependency too much.

Reply:
Yes, I like the idea of keeping the number of deps small (I could have noticed that at the first time...).
At the end, it will be a rewrite of `mkdir_p` function. Might take another day or two (got busy with another task).

Reply:
OK, this should now be better: No additional dependency, just relying on the `astring `package (which is already a dep). 358a4750a06389526b53f6bd93520e0c5313e6ca